jax/numpy/lax_numpy.py

```diff
@@ -2609,6 +2609,20 @@ def polyval(p, x):
     y = y * x + p[i]
   return y
 
+@_wraps(np.polyadd)
+def polyadd(a, b):
+  if isinstance(a, np.poly1d):
+    a = np.asarray(a)
+  if isinstance(b, np.poly1d):
+    b = np.asarray(b)
```

Comment:
These `isinstance` checks and ``np.asarray`` calls mean the function can't be jit-compiled. I believe that it could be replaced with
```
a = jnp.array(a)
b = jnp.array(b)
```